### DISCUSSION TITLE
Is there any information about the first stable release v1

### DISCUSSION THREAD

**Discussion by vikingair** [vitest-dev/vitest#3011] [Q&A]
Sorry, I couldn't find any hint on this. I have the feeling that the current version is already mature enough to be released under a stable version. I would like to recommend officially as replacement over Jest.

So can anybody share some insights on the roadmap to v1?

**Comment by krichards** [comment]
This is related to https://github.com/vitest-dev/vitest/discussions/3476. A 1.0.0 release would help to start getting updates easily.

**Comment by sheremet-va** [comment]
Before 1.0 we need to deprecate some API (`deps.inline`/`deps.registerNodeLoader`) and make sure `deps.optimizer` can safely replace them.

Other blockers are `vm` support to run tests faster and a new strict Node mode (currently Vitest injects `module` and `exports` to _any_ file, which is unsafe when migrating from CJS to ESM).

The current plan is to release Vitest 1.0 no earlier than September 11th (EOL for Node 16). This however can change if features mentioned earlier are not implemented by that time.
